bom/runtime/pom.xml

```diff
@@ -893,6 +889,11 @@
                         <groupId>com.sun.xml.bind</groupId>
                         <artifactId>jaxb-osgi</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <!-- https://github.com/quarkusio/quarkus/issues/294 -->
```

Comment:
I would remove that. If we add a reference every time we add some new code, it will get unreadable soon.

Reference the GitHub issue in your commit message and then blame is your friend.